plugin/clang.vim

```diff
@@ -840,10 +840,27 @@ func! s:ClangCompleteInit(force)
 endf
 "}}}
 "{{{ ClangExecuteNeoJobHandler
-"handles event: exit
+"Handles events: stdout, stderr, exit
 func! ClangExecuteNeoJobHandler(job_id, data, event)
-  if a:event == 'exit'
-    call ClangExecuteDone(self.fstdout, self.fstderr)
+  if a:event == 'stdout' || a:event == 'stderr'
+    if len(b:clang_state[a:event]) != 0
+      if b:clang_state[a:event][-1] == ''
+        call remove(b:clang_state[a:event], -1)
+      else
+        let line = a:data[0]
+        let b:clang_state[a:event][-1] .= line
+        call remove(a:data, 0)
+      endif
+    endif
+    let b:clang_state[a:event] += a:data
+  else
```

Comment:
Seems an empty line at the end of stdout and stderr can be omitted automatically by plugin, so there's no need to remove the last empty line?